docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -6,6 +6,10 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 = {project-name} - Contexts and Dependency Injection
+:numbered:
```

Comment:
are you sure it won't break the website if you change this embedded file?
Better double check

Reply:
Extension guide has the same TOC config. So it should not break the website. But to be honest, we should add some styles to the TOC on the website (something like "float:left;") because the TOC is sometimes too long and the guide does not read well.

Reply:
https://github.com/quarkusio/quarkusio.github.io/issues/196